client/src/components/ResourceInternalNotesForm.jsx

```diff
@@ -85,7 +84,8 @@ const InternalNotesForm = Form.create({ name: 'internalNotes' })(
           })(
             <Input
               placeholder="Subject"
-              defaultValue={!editNote == null && editNote.body}
+              // eslint-disable-next-line @hack4impact-uiuc/no-null-ternary
```

Comment:
what is this comment?

Reply:
it's to disable linting errors (if you don't have it the app will scream at you)

Reply:
yeahhh if it wasn't here then the linter would correct it so it's broken (I should honestly talk to @arpanlaha about this on why it's breaking)

Reply:
next line should be `editNote?.body`